ksql-parser/src/test/java/io/confluent/ksql/parser/tree/TableElementsTest.java

```diff
@@ -112,26 +115,6 @@ public void shouldThrowOnDuplicateValueColumns() {
     TableElements.of(elements);
   }
 
-  @Test
-  public void shouldThrowOnDuplicateKeyValueColumns() {
```

Comment:
Why has this test been removed?  Bad things happen if we allow duplicates at the moment. We really don't want to allow this!

If this is a requirement for the parsing of logical schemas, then it once again suggests that we need two different parsers: one of logicl schemas and one for CT/CS schemas.